src/utils/1inch.js

```diff
@@ -26,8 +26,6 @@ import Toast from 'components/Toast';
 import { convertToBaseUnits, reportLog } from 'utils/common';
 
 const EXCHANGE_ETH_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
-export const EXCHANGE_URL = 'https://api.1inch.exchange/v1.1';
-export const EXCHANGE_ADDRESS = '0xe4c9194962532feb467dce8b3d42419641c6ed2e';
```

Comment:
These are still imported in `services/1inch`, I don't think you can just delete them?

Reply:
Gosh, my IDE fucked up imports, it was marked as unused, sorry for confusion! 😱